tests/yam/agama/import_agama_profile.pm

```diff
@@ -18,6 +25,8 @@ sub run {
 
     select_console 'root-console';
     assert_script_run("agama profile import $profile_url", timeout => 300);
+    # start headless install
+    background_script_run('agama install > /dev/null 2>&1') if (get_var('EXTRABOOTPARAMS', '') =~ /systemd.unit=multi-user.target/);
```

Comment:
what I didn't get is why we needed to invent any new mechanism with this, I was trying to following your implementation and then I recommended to use the puppeteer module that monitor the installation, but why not just use the default installation module? it would be more simple. Then you don't need to modify this module.

Reply:
I just don't know at this point the install was start or not, I'll try to remove this and use puppeteer this afternoon.

Reply:
you don't need to run unattended installation, with default puppeteer test module you just start the installation with puppeteer, anyway puppeteer always open a new browser.

Reply:
it [works](https://openqa.suse.de/tests/17500101#) and updated PR and rerun all VRs.